docs/tutorials/Upgrading_to_Application_Commands.md

```diff
@@ -0,0 +1,309 @@
+
+![Client interfaces showing the different types of application commands](command-types.png)
+
+> info
+> If you are developing an app for the first time, the [commands documentation](#DOCS_INTERACTIONS_APPLICATION_COMMANDS) may be a more helpful resource for you.
+
+Before diving in, it’s worth noting that with the message content changes, apps can still access message content in their DMs with users, as well as when messages are sent that directly `@mentions` their bot user (since there is clear user intent).
+
+And for apps that use message content in ways that can’t be ported to commands, you should review the [message content intent review policy](https://support-dev.discord.com/hc/en-us/articles/5324827539479) to see if you’re eligible to apply for the privileged intent.
+
+---
+
+## Types of commands
+
+There are three types of application commands: slash commands, user commands, and message commands. When building your app, whether migrating or starting from scratch, consider which type of command(s) makes sense for a given feature of your app.
+
+### Slash commands
+
+[Slash commands](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/slash-commands) are the most common type of command. They are accessed by typing the command’s name prefixed with a backslash (`/`) or by using the plus button (+) to the left of the message input.
+
+Slash commands can appear in channels and DMs, so they’re helpful when an action is tied to a channel, a server, or nothing at all.
+
+### User commands
+
+[User commands](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/user-commands) are in the context menu for users, which is accessed by right-clicking or tapping on any user. They’re helpful when an action is tied to an individual user, like a moderation app adding a timeout to someone.
+
+> info
+> Within the context menus for users and messages, the `Apps` submenu will only appear if an app in that server has installed a corresponding command (whether or not an individual user can use the installed command).
+
+### Message commands
+
+[Message commands](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/message-commands) are in the context menu for messages, which is accessed by right-clicking on a message, or by clicking on the ellipses at the top-right of a message. They’re helpful when an action is tied to a message, like creating a reminder to reply to a message the following day.
+
+### After this section
+- Start thinking about how different app features might map to the different types of application commands.
+
+## Registering commands
+
+Commands can be registered via HTTP requests after an app is authorized with the `applications.commands` scope. Since commands aren’t tied to bot users, being authorized with the `bot` scope isn’t sufficient. Any commands created before an app is added to a server will automatically be installed.
```

Comment:
This implies that you need to send a request so that global commands will populate in a guild

Reply:
(whoops, didn't read the whole paragraph)